tests/mock_crust/churn.rs

```diff
@@ -131,6 +150,15 @@ impl ExpectedGets {
                 }
             }
         }
+        while let Ok(event) = clients[0].inner.try_next_ev() {
```

Comment:
There shouldn't really be any harm in testing all the clients in the slice here I think.  Not a big deal, but it would make the test more thorough if we assert that none of the other clients received any messages by mistake.